opensoundscape/utils.py

```diff
@@ -329,3 +330,25 @@ def generate_opacity_colormaps(
         colormaps.append(cmap)
 
     return colormaps
+
+
+import numpy as np
```

Comment:
imports should always be at the top of a file. VS Code will help detect duplicate imports

Reply:
Oh, sorry, missed that. Just pushed the correction.